src/OpenTK.GraphicsLibraryFramework/Interfaces/IGLFW.cs

```diff
@@ -279,144 +380,202 @@ internal interface IGLFW
+        /// </para>
         /// </summary>
         /// <param name="monitor">The monitor to query.</param>
         /// <returns>The current gamma ramp, or <c>null</c> if an error occurred.</returns>
         /// <remarks>
+        /// <para>
         /// The returned structure and its arrays are allocated and freed by GLFW.
         /// You should not free them yourself. They are valid until the specified monitor is disconnected,
         /// this function is called again for that monitor or the library is terminated.
-        ///
+        /// </para>
+        /// <para>
         /// This function must only be called from the main thread.
-        ///
-        /// TODO: Possible errors include <see cref="ErrorCode.NotInitialized"/> and <see cref="ErrorCode.PlatformError"/>.
+        /// </para>
+        /// <para>
+        /// Possible errors include <see cref="ErrorCode.NotInitialized"/> and <see cref="ErrorCode.PlatformError"/>.
+        /// </para>
         /// </remarks>
         unsafe GammaRamp* GetGammaRamp(Monitor* monitor);
 
         /// <summary>
+        /// <para>
         /// This function sets the current gamma ramp for the specified monitor.
+        /// </para>
+        /// <para>
         /// The original gamma ramp for that monitor
         /// is saved by GLFW the first time this function is called and is restored by <see cref="Terminate"/>.
+        /// </para>
         /// </summary>
         /// <param name="monitor">The monitor whose gamma ramp to set.</param>
         /// <param name="ramp">The gamma ramp to use.</param>
         /// <remarks>
+        /// <para>
         /// Gamma ramp sizes other than 256 are not supported by all platforms or graphics hardware.
+        /// </para>
+        /// <para>
         /// Windows: The gamma ramp size must be 256.
-        ///
+        /// </para>
+        /// <para>
         /// The specified gamma ramp is copied before this function returns.
-        ///
+        /// </para>
+        /// <para>
         /// This function must only be called from the main thread.
-        ///
-        /// TODO: Possible errors include <see cref="ErrorCode.NotInitialized"/> and <see cref="ErrorCode.PlatformError"/>.
+        /// </para>
+        /// <para>
+        /// Possible errors include <see cref="ErrorCode.NotInitialized"/> and <see cref="ErrorCode.PlatformError"/>.
+        /// </para>
         /// </remarks>
         unsafe void SetGammaRamp(Monitor* monitor, ref GammaRamp ramp);
 
         /// <summary>
+        /// <para>
         /// This function resets all window hints to their default values.
+        /// </para>
         /// </summary>
         /// <remarks>
+        /// <para>
         /// This function must only be called from the main thread.
+        /// </para>
         /// </remarks>
-        /// <seealso cref="WindowHint(WindowHintInt,int)"/>
         void DefaultWindowHints();
 
-        // TODO: not found in GLFW? void WindowHintString(int hint, string value);
```

Comment:
That one was my bad. It is found in 3.3.

Reply:
Will re-add later.

Reply:
Fixed, this is now added.